modules/sdk-coin-btc/src/inscriptionBuilder.ts

```diff
@@ -62,16 +62,16 @@ export class InscriptionBuilder implements IInscriptionBuilder {
       signer = 'user',
       cosigner = 'bitgo',
       inscriptionConstraints = DefaultInscriptionConstraints,
-      changeAddressType = 'p2tr',
+      changeAddressType = 'p2wsh',
     }: {
-      signer: utxolib.bitgo.KeyName;
-      cosigner: utxolib.bitgo.KeyName;
-      inscriptionConstraints: {
+      signer?: utxolib.bitgo.KeyName;
```

Comment:
why these optional? Just not needed for p2wsh but were for p2tr?

Reply:
I had to pass in all those parameters because they were not optional even though they have default values